index.js

```diff
@@ -22,7 +22,11 @@ exports.init = function(opts) {
   exports.SchemaAdapter = require('./adapters/mongoose');
 
   exports.getModels = function () {
-    return opts.mongoose.models;
+    if ((typeof opts.getModels) === 'function') {
```

Comment:
Yes both `includedModels` and `excludedModels` would be great.
The idea is to create a function to replace all `opts.mongoose.models` calls, yes 👍